projects/plugins/jetpack/extensions/blocks/payments-intro/editor.scss

```diff
@@ -1,17 +1,53 @@
 @import '@automattic/jetpack-base-styles/gutenberg-base-styles';
 
 .wp-block-jetpack-payments-intro {
-	.components-placeholder__instructions {
-		// Defer to vertical margins applied by template picker options.
-		margin-bottom: 0;
-	}
```

Comment:
I think this is causing Jetpack sites to show more space now below instructions

Before | After
--- | ---
<img width="679" alt="Screen Shot 2022-07-21 at 12 04 50" src="https://user-images.githubusercontent.com/1233880/180188503-12a0e088-6cce-4fcd-870f-37579b95120e.png"> | <img width="702" alt="Screen Shot 2022-07-21 at 12 04 10" src="https://user-images.githubusercontent.com/1233880/180188524-a9ff650d-80d3-4ea9-aa99-8f0984442394.png">